site/docs/plugins/i18n.md

```diff
@@ -409,7 +421,7 @@ bot.use(
   }),
 );
 
-// Register the i18n middleware
+// Register the i18n middleware after session middleware
```

Comment:
This change is not related to the PR's goal

```suggestion
// Register the i18n middleware
```

Reply:
I'm sorry, my bad.